js/classes/entities/midground/enemies/flying_enemy_base.js

```diff
@@ -123,4 +134,19 @@ class FlyingEnemyBase {
         return [new Vector(0, 0), new Vector(200, 0), new Vector(0, 0)];
     }
 
+    static get JUMP_SMALL() {
```

Comment:
What is the purpose of these jump functions? Do flying enemies jump?

Reply:
yeah, it's kind of strange, but they do jump. GroundEnemyBase doesn't support jumping to that extent, so I need to find a quick work-around.